app/graphql/mutations/create_billing_customer.rb

```diff
@@ -27,10 +27,8 @@ def resolve(input:)
                 zip_code: input.zip_code
             )
 
-            phone_number, tax_exempt = input.values_at(:phone_number, :tax_exempt)
-
-            new_billing_customer.phone_number = phone_number if phone_number
-            new_billing_customer.tax_exempt = tax_exempt if tax_exempt
+            new_billing_customer.phone_number = input.phone_number if input.phone_number
```

Comment:
I changed this because apparently the `input` argument in the mutation isn't a typical Ruby hash, so the `values_at` method doesn't work as intended...